cli/get-multidomain-certificate.md

```diff
@@ -0,0 +1,64 @@
+---
+currentMenu: cli-get-multidomain-certificate
+---
+
+# Multi-domains certificates
+
+Let's Encrypt and most ACME servers are able to provide multi-domain certificates. Such certificates
+will be usable for multiple domains as a single file, which can be useful in many cases (for instance to
+use the same certificate for `yourdomain.org` and `www.yourdomain.org`).
+
+**Note:** Multi-domains certificates use a technology called [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication)
```

Comment:
Not exactly: In SNI, the client send the hostname ans the server _select_ the best certificate who fit with the domain.

The multi-domains certificate generate (as you describe very well) 1 certificate with several subjects. The server does not have to _choose_ a certificate, it just send the big certificate which contains every thing.

SANs is pretty old and well supported by browsers (https://www.digicert.com/subject-alternative-name-compatibility.htm)